### DISCUSSION TITLE
Allow defining a theme-level outline style configuration?

### DISCUSSION THREAD

**Discussion by Nantris** [mantinedev/mantine#2825] [Feature requests]
We want to override the default `outline-width: 2px` with `outline-width: 0.125rem` without hard to work with resorting to global styles that use `!important`, but there doesn't seem to be any way to achieve this besides specifying it in every individual element.

It would be great to be able to customize the focus-ring.

The rationale for this is that our application allows users to change the base font size, and when they increase or decrease the font size in the app, where everything else is coded with `em` or `rem`, the hardcoded `2px` value can appear too small or too wide. The same issue with the default `outline-offset`.

Is there any way around this that doesn't involve global `!important` or manually assigning the style to every element?

Similar to, but distinct from, #2789.

**Comment by rtivital** [comment]
It is panned to add a theme property to override focus styles

**Reply by Nantris** [reply]
Thanks so much for your reply @rtivital - looking forward to it.

**Reply by rtivital** [reply]
Already implemented, will be available with 5.7 release
